site/content/installation/installing-nic/installation-with-helm.md

```diff
@@ -13,6 +13,8 @@ This document explains how to install F5 NGINX Ingress Controller using [Helm](h
 
 {{< note >}} All documentation should only be used with the latest stable release, indicated on [the releases page]({{< relref "releases.md" >}}) of the GitHub repository. {{< /note >}}
 
+{{< note >}} From release 4.0.1 onwards Helm chart location has changed from oci://ghcr.io/nginxinc/charts/nginx-ingress to oci://ghcr.io/nginx/charts/nginx-ingress. {{< /note >}}
```

Comment:
```suggestion
{{< warning >}} From release 4.0.1 onwards, the Helm chart location has changed from `oci://ghcr.io/nginxinc/charts/nginx-ingress` to `oci://ghcr.io/nginx/charts/nginx-ingress`. {{< /warning >}}
```

Reply:
We should probably think about a formal timeline for removing information that's specific to a version that might not be relevant to users in the future. Some of the NGINX products have a convention of removing resolved issues after 3 releases, for example.

It'll be weird to see this notice in the document at the point it's at version 5.x or 6.x: I genuinely think the release page is enough, since that's where a user upgrading will look to know any changes.

Reply:
thanks @ADubhlaoich, I'll just keep it on Releases page 👍🏼